### PR TITLE
Update Privacy policy

### PR DESCRIPTION
**Pull request by titlescreen**
DPIA have looked at the privacy policy and recommended some changes.
The 'Why our use of your personal data is lawful' section still mentions
consent which feels off. We are currently awaiting feedback on that
sentence.

### CODE REVIEW

File: app/views/static_pages/privacy_policy.html.erb [hunk 1]
```diff
@@ -26,16 +26,16 @@
...
-      <li>as a user of the service</li>
     </ul>
 
     <h2 class="govuk-heading-l">
       What personal information we will collect
     </h2>
 
     <p class="govuk-body">
-      We will collect:
+      If you make a claim we will collect your personal information such as your:
```
Comment (erbridge) [inline app/views/static_pages/privacy_policy.html.erb]: The second bullet point is unrelated to the user making a claim.

File: app/views/static_pages/privacy_policy.html.erb [hunk 2]
```diff
@@ -112,14 +112,14 @@
 
     <p class="govuk-body">
       Neither DfE nor its delivery partners will keep your personal data longer than we need it for the purpose(s) of
-      research, security and/or delivery of the service, and in any case not longer than XXXX years. Your data will be
-      deleted when DfE and its delivery partners no longer need it for these purposes, or when XXXX years have passed,
+      research, security and/or delivery of the service, and in any case not longer than 7 years. Your data will be
+      deleted when DfE and its delivery partners no longer need it for these purposes, or when 7 years have passed,
```
Comment (erbridge) [inline app/views/static_pages/privacy_policy.html.erb]: Can we actually say 7 years? Does that mean people will be able to reclaim after 7 years?

File: app/views/static_pages/privacy_policy.html.erb [hunk 3]
```diff
@@ -112,14 +112,14 @@
...
       whichever comes first.
     </p>
 
     <p class="govuk-body">
-      Partial IP addresses will be deleted after XXXX days unless their request caused an error, in which case they remain
-      in our logs for XXXX days.
+      Partial IP addresses will be deleted after 7 days unless their request caused an error, in which case they remain
+      in our logs for 30 days.
```
Comment (erbridge) [inline app/views/static_pages/privacy_policy.html.erb]: IIRC, TVS did something clever with munging the IPs to they only stored partial ones, but I don't remember the details. We don't store any IPs at all currently, and we're not set up to make this statement at all true if we did start to collect them. Can we just remove it?

File: app/views/static_pages/privacy_policy.html.erb (hunk 2 above)
Comment (titlescreen) [reply app/views/static_pages/privacy_policy.html.erb]: Checked and we can remove the reference to the dates.

File: app/views/static_pages/privacy_policy.html.erb [hunk 4]
```diff
@@ -62,9 +62,10 @@
     </h2>
 
     <p class="govuk-body">
-      In order for our use of your personal data to be lawful, we need to meet one (or more) conditions in the data
-      protection legislation. For the purpose of this project, your consent forms the basis of this as stated under GDPR
-      Article 6 (1)(a).
+      In order for our use of your personal data to be lawful, we need to meet one or more conditions in the data
+      protection legislation.  For the purpose of ‘<%= t("service_name") %>‘, the Department processes your
```
Comment (erbridge) [inline app/views/static_pages/privacy_policy.html.erb]: Second curly quote is backwards. Not sure it matters.

Comment (tekin) [reply app/views/static_pages/privacy_policy.html.erb]: yep, should be `’`

**Pull request merged**